File: lib/javascript/001_solution.js
var aTimer = process.hrtime();
var sum = 0;
for (var x = 3 ; x < 100000000 ; x++) {
  if (x % 3 === 0 || x % 5 === 0) {
    sum += x;
  }
}
console.log('The sum of them is: '+ sum);
var aTimerDiff = process.hrtime(aTimer);
console.info("Execution time: %dseconds", aTimerDiff[1]/1000000000);

File: lib/ruby/001_solution.rb
# Find the sum of all the multiples of 3 or 5 below 1000.
require 'benchmark'
total = 0
execution_time = Benchmark.realtime{  
  x = 3
  until x >= 100000000
    if x % 3 == 0 || x % 5 == 0
      total += x
    end
    x += 1
  end}
puts "The sum is #{total}"
puts "Execution Time: #{"%1.12f" % execution_time} seconds"